tests/testInclude.py

```diff
@@ -0,0 +1,131 @@
+
+try:
+    type(display)
+except NameError:
+    display = False
+
+DATA_DIR = os.path.join(os.environ["MEAS_DEBLENDER_DIR"], "tests", "data")
+
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+class IncludeTestCase(lsst.utils.tests.TestCase):
+    """ Test case for DM-1738: test that include method successfully
+    expands footprint to include the union of itself and all others provided.
+
+    In the current version of meas_deblender, children's Footprints can extend outside
+    that of the parent.  The replace-by-noise code thus fails to reinstate the pixels
+    that are outside the parent but within the children.  Calling include() prior
+    to noise replacement solves this issue.
+
+    The test data used here (./data/ticket1738.fits) is a cutout of an HSC observed M31
+    field in which this pathology occured:
+    --id field=M31 dateObs=2014-11-24 visit=14770 ccd=27
+    bbox = afwGeom.Box2I(afwGeom.Point2I(720, 460), afwGeom.Extent2I(301, 301))
+
+    See data/ticket1738_noInclude.png or data/ticket1738_noInclude.fits for a visual
+    of this pathology).
+
+    """
+
+    def setUp(self):
+        self.calexpOrig = afwImage.ExposureF(os.path.join(DATA_DIR, "ticket1738.fits"))
+        self.calexp = afwImage.ExposureF(os.path.join(DATA_DIR, "ticket1738.fits"))
+        self.psf = self.calexp.getPsf()
+
+    def tearDown(self):
+        del self.calexpOrig
+        del self.calexp
+        del self.psf
+
+    def testInclude(self):
+        schema = afwTable.SourceTable.makeMinimalSchema()
+
+        # Create the detection task
+        config = SourceDetectionTask.ConfigClass()
+        config.thresholdPolarity = "both"
+        config.thresholdValue = 5
+        config.reEstimateBackground = False  # Turn off so that background does not change from orig
+        detectionTask = SourceDetectionTask(config=config, schema=schema)
+
+        # Create the deblender Task
+        debConfig = measDeb.SourceDeblendConfig()
+        debConfig.maxNumberOfPeaks = 10
+        debTask = measDeb.SourceDeblendTask(schema, config=debConfig)
+
+        # Create the measurement Task
+        config = SingleFrameMeasurementTask.ConfigClass()
+        measureTask = SingleFrameMeasurementTask(schema, config=config)
+
+        # Create the output table
+        tab = afwTable.SourceTable.make(schema)
+
+        # Process the data
+        result = detectionTask.run(tab, self.calexp)
+
+        sources = result.sources
+
+        # Run the deblender
+        debTask.run(self.calexp, sources, self.psf)
+        # Run the measurement task
+        measureTask.run(sources, self.calexp)
+
+        plotOnFailure = False
+        if display:
+            plotOnFailure = True
+
+        self.assertClose(self.calexpOrig.getMaskedImage().getImage().getArray(),
+                         self.calexp.getMaskedImage().getImage().getArray(),
+                         atol=1E-3, printFailures=False, plotOnFailure=plotOnFailure)
```

Comment:
Why this value for `atol`? (I guess "because this makes the test pass", but it's nice to be explicit -- any time there's a number baked into the code, it makes me wonder where it came from and if it's in some way special.)